Framework/script/RepoCleaner/qcrepocleaner/Ccdb.py

```diff
@@ -57,15 +57,15 @@ def __init__(self, url):
         logger.info(f"Instantiate CCDB at {url}")
         self.url = url
 
-    def getObjectsList(self, added_since: int = 0) -> List[str]:
+    def getObjectsList(self, added_since: int = 0, path: str = "") -> List[str]:
         '''
         Get the full list of objects in the CCDB that have been created since added_since.
 
         :param added_since: if specified, only return objects added since this timestamp in epoch milliseconds.
         :return A list of strings, each containing a path to an object in the CCDB.
         '''
         logger.debug(f"added_since : {added_since}")
-        url_for_all_obj = self.url + '/latest/.*'
+        url_for_all_obj = self.url + '/latest/' + path + '/.*'
```

Comment:
if the path is empty, we get double "/", is this fine?

Reply:
yes, I tested this case